app/github_public_repo/README.md

```diff
@@ -0,0 +1,26 @@
+
+# SARS-CoV-2 Serosurveillance Data by Serotracker
+
+[SeroTracker](https://serotracker.com/en/Explore) synthesizes findings from thousands of COVID-19 seroprevalence studies worldwide, providing a data platform and interactive dashboard for SARS-CoV-2 serosurveillance. This dataset in this repository represents our collection of serosurveillance studies.
+
+### Download
+Our complete SARS-Cov-2 dataset can be scraped programmatically from this repository in the following ways:
+1. Using the command: `curl https://raw.githubusercontent.com/serotracker/sars-cov-2-data/main/test.csv`.
+2. Using a GET request through REST API protocol.
+
+Our dataset can also be downloaded directly from our [Airtable view]([https://airtable.com/shraXWPJ9Yu7ybowM/tbljN2mhRVfSlZv2d?backgroundColor=blue&viewControls=on](https://airtable.com/shraXWPJ9Yu7ybowM/tbljN2mhRVfSlZv2d?backgroundColor=blue&viewControls=on)).
+
+The CSV file follows the format of 1 row per serosurveillance study.
```

Comment:
Is this the case? Are we only pushing primary estimates from each study? Or are we pushing one row per estimate, and each row is one seroprevalence estimate - that is, are we making subgroup data available?

Reply:
No, this file is wrong. One row corresponds to one estimate. We are showing all estimates not just primary estimates. Need to update this.